apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx

```diff
@@ -67,8 +67,16 @@ type LayoutProps = {
 };
 
 const Layout = ({ category, cards }: LayoutProps) => {
-  // TODO : handle cards impressions with logImpressionCard
-  const { logClickCard, dismissCard, trackContentCardEvent } = useDynamicContent();
+  const { logClickCard, dismissCard, trackContentCardEvent, logImpressionCard } =
+    useDynamicContent();
+
+  useEffect(() => {
+    logImpressionCard(category.id);
+    for (const card of cards) {
+      logImpressionCard(card.id);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please do not "lie" to React: https://react.dev/learn/removing-effect-dependencies#why-is-suppressing-the-dependency-linter-so-dangerous

Reply:
Looking at the doc, while useEffectEvent isn't stable yet, I don't see how I can run a function at the component mount without having to mute the linter 🤔